src/coffee/mapping-utils/csv.coffee

```diff
@@ -76,18 +96,37 @@ class CsvMapping
       return code
     ).join(';')
   # TODO: Move method below to sphere-node-utils
+
+  formatProductAttributes: (attributeName, attributes) ->
+    mappedAttributes = @productMapper._mapAttributes(attributes)
+    value = mappedAttributes[attributeName]
+
+    if value is undefined
+      value = ''
+    value
+
   formatPrice = (price) ->
-    if price?.value?
-      countryPart = ''
-      if price.country?
-        countryPart = "#{price.country}-"
-      customerGroupPart = ''
-      if price.customerGroup?
-        customerGroupPart = " #{price.customerGroup.id}"
-      channelKeyPart = ''
-      if price.channel?
-        channelKeyPart = "##{price.channel.id}"
-      "#{countryPart}#{price.value.currencyCode} #{price.value.centAmount}#{customerGroupPart}#{channelKeyPart}"
+#    if price?.value?
```

Comment:
Should all these commented lines be here?

Reply:
Removed here 867eadc